develop/smart-contracts/local-development-node.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Local Development Node
+description: Set up a local node for smart contract development. Follow this step-by-step guide to install the substrate node & ETH-RPC for testing in a local environment.
+---
+
+# Local Development Node
+
+## Introduction
+
+A local development node provides an isolated blockchain environment where you can deploy, test, and debug smart contracts without incurring network fees or waiting for block confirmations. This guide demonstrates how to set up a local Polkadot SDK-based node with smart contract capabilities.
+
+By the end of this guide, you'll have:
+
+- A running substrate node with smart contract support
+- An ETH-RPC adapter for Ethereum-compatible tooling integration accessible at `http://localhost:8545`
+
+## Prerequisites
+
+Before getting started, ensure you have done the following:
+
+- Completed the [Install Polkadot SDK Dependencies](/develop/parachains/install-polkadot-sdk/){target=\_blank} guide and successfully installed [Rust](https://www.rust-lang.org/){target=\_blank} and the required packages to set up your development environment
+
+## Installation
```

Comment:
Install what? Ideally this is a little bit more specific and since we're instructing users, it should be action oriented. Not sure if there's a better way to phrase this, but maybe "Install the Substrate Node and ETH-RPC Adapter"